example/feature_demo/demo_types.proto

```diff
@@ -50,6 +83,9 @@ message MultiaccountTypeWithID {
 }
 
 message MultiaccountTypeWithoutID {
+  // here no id field is given, but it is still a multitenant type
+  // this is a bad pattern, as there is no way to uniquely identify objects
+  // in the DB, so R/U/D behavior is not well defined
```

Comment:
will this cause generation, or result in no GORM model for the type?

Reply:
Currently it will generate, and but will cause issues compile time once associations get involved. Should we require that there is a field named 'id' or with a gorm tag of 'primary_key'? If yes, should that be a plugin failure condition, or just a 'drop this object' condition?